frontend/src/routes/(authenticated)/admin/EditUserAccount.svelte

```diff
@@ -120,6 +123,21 @@
     error={errors.role}
     disabled={_user.id === currUser.id}
   />
+  <AdminContent>
+    <div>
+      Feature flags:
+      <ul>
+        {#each allPossibleFlags as flag}
+          <li><input
+            type="checkbox"
+            name="featureFlags"
+            value={flag}
+            bind:group={_user.featureFlags}
+          > {flag}</li>
```

Comment:
when I played around in the UI I found the label was not clickable. Could we do something about that? I think we need to make a label and connect it to the input to make that work well. Thanks

Reply:
Yep, that was quick-and-dirty and I also figured out that the label should be clickable. I also should be turning this into a proper form field (using `$form.featureFlags`) and use the Checkbox component for better visual integration with the rest of the UI.

Reply:
Done in commit 1a73ef4b.